gdx/src/com/badlogic/gdx/math/collision/OrientedBoundingBox.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.math.collision;
+
+import com.badlogic.gdx.math.Matrix4;
+import com.badlogic.gdx.math.Vector3;
+
+import java.io.Serializable;
+
+public class OrientedBoundingBox implements Serializable {
```

Comment:
did you tested with GWT? because it needs to be referenced, see https://github.com/libgdx/libgdx/blob/master/gdx/res/com/badlogic/gdx.gwt.xml#L370

Reply:
Nice catch! I didn't, I forgot where this file was and couldn't find using search. Thanks!